app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -247,17 +247,23 @@ def doc_class_name
           true_id_product&.dig(:AUTHENTICATION_RESULT, :DocClassName)
         end
 
+        def doc_issuer_type
+          true_id_product&.dig(:AUTHENTICATION_RESULT, :DocIssuerType)
+        end
+
```

Comment:
This is a complete nit, but could you please change "Acuent" to "Acuant" in line 255? Thanks!